web/src/screens/filters/action.tsx

```diff
@@ -411,6 +411,42 @@ const TypeForm = ({ action, idx, clients }: TypeFormProps) => {
         />
       </div>
     );
+  case "PORLA":
+    return (
+      <div className="w-full">
+        <div className="mt-6 grid grid-cols-12 gap-6">
+          <DownloadClientSelect
+            name={`actions.${idx}.client_id`}
+            action={action}
+            clients={clients}
+          />
+
+          <div className="col-span-6 sm:col-span-6">
+            <TextField
+              name={`actions.${idx}.save_path`}
+              label="Save path"
+              columns={6}
+              placeholder="eg. /full/path/to/watch_folder"
```

Comment:
Update placeholder to `/full/path/to/torrent/data` or similar. Probably copied from existing code that should be updated but that's for another pr :)